resources/shaders/deferred_composition.hlsl

```diff
@@ -63,10 +67,10 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 	if(depth_f != 1.0f)
 	{
 		// GBuffer contents
-		float3 albedo = gbuffer_albedo_roughness[screen_coord].xyz;
-		const float roughness = gbuffer_albedo_roughness[screen_coord].w;
-		float3 normal = gbuffer_normal_metallic[screen_coord].xyz;
-		const float metallic = gbuffer_normal_metallic[screen_coord].w;
+		float3 albedo = gbuffer_albedo_roughness.SampleLevel(point_sampler, uv, 0).xyz;
```

Comment:
Why sampling from a gbuffer?

Reply:
thats just the buffer names, i changed them from using SampleLevel instead of direct array access for scaling resolutions support.